lib/versionbot.ts

```diff
@@ -389,7 +389,7 @@ export class VersionBot extends GithubBot.GithubBot {
             // Now we get the new version from the CHANGELOG (*not* the package.json, it may not exist).
             return exec(`cat ${versionData.fullPath}${_.last(moddedFiles)}`).then((contents: string) => {
                 // Only interested in the first match for '## v...'
-                const match = contents.match(/^## (v[0-9]\.[0-9]\.[0-9]).+$/m);
+                const match = contents.match(/^## (v[0-9]+\.[0-9]+\.[0-9]+).+$/m);
```

Comment:
Ok cool, it'd be great if versionist could help here so that, in case we change `versionist.conf.js`, we don't have to remember changing the fixed version parsing logic here as well. (Just a note, I don't think we should block here for the first version, so lgtm)